packages/block-library/src/post-template/index.php

```diff
@@ -82,17 +82,28 @@ function render_block_core_post_template( $attributes, $content, $block ) {
 	$content = '';
 	while ( $query->have_posts() ) {
 		$query->the_post();
+
+		// Get an instance of the current Post Template block.
+		$block_instance = $block->parsed_block;
+
+		// Set the block name to one that does not correspond to an existing registered block.
+		// This ensures that for the inner instances of the Post Template block, we do not render any block supports.
+		$block_instance['blockName'] = 'core/null';
+
+		// Render the inner blocks of the Post Template block, with `dynamic` set to `false` to ensure that no wrapper markup is included.
```

Comment:
```suggestion
		// Render the inner blocks of the Post Template block with `dynamic` set to `false` to prevent calling 
 render_callback and ensure that no wrapper markup is included.
```

This is a good comment! Saves folks searching around for API docs. Would it be worth adding that `'dynamic' => false` skips `render_callback` to explain the how and the what?

Reply:
Oh, good suggestion! Yes, I think because of how unusual this block is, the more explanatory text, the better, to hopefully make it a little easier for anyone making future changes. I'll tweak the wording a little later today 🙂

Reply:
Committed in https://github.com/WordPress/gutenberg/pull/41827/commits/c8e28148155bf314ba9905db8537fa818480d826. Thanks!